legacy/app/auth/login.html

```diff
@@ -6,19 +6,19 @@
 
         <div class="form-field">
             <label for="email" translate>user.email</label>
-            <input type="email" id="email" ng-model="email" ng-disabled="processing" placeholder="name@domain.com" required>
+            <input type="email" id="email" ng-model="email" ng-disabled="processing" placeholder="name@domain.com" required data-cy="email">
         </div>
 
         <div class="form-field">
             <label for="password" translate>user.password</label>
-            <input id="password" type="password" ng-model="password" ng-disabled="processing" required>
+            <input id="password" type="password" ng-model="password" ng-disabled="processing" required data-cy="password">
```

Comment:
change `data-cy=password` to `data-cy=password-field`?

Reply:
👍